src/ATen/function_wrapper.py

```diff
@@ -58,14 +70,6 @@
     return static_cast<Type*>(this)->${method_prefix}${api_name}(${zero_dim_actuals});
 }""")
 
-SCALAR_EXPAND = CodeTemplate("""\
```

Comment:
Can you explain why `SCALAR_EXPAND` was removed?

Reply:
The expand_X code does this now.